src/ui/components/Timeline/index.tsx

```diff
@@ -260,7 +282,7 @@ class Timeline extends Component<PropsFromRedux> {
   renderUnloadedRegions() {
     const { loadedRegions, zoomRegion } = this.props;
 
-    if (!loadedRegions || loadedRegions.length === 0) {
+    if (!loadedRegions || loadedRegions.length === 0 || !this.isFinishedLoadingRegions()) {
```

Comment:
These first two conditions can be dropped since they are covered by the new function.